docs/UserGuide.adoc

```diff
@@ -66,11 +66,12 @@ Examples:
 
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `a n/Nam p/90234 e/n@example.com a/abc t/me`
```

Comment:
I think it is better to have a more valid example here, eg. with a proper name and surname, as we may add data validation later and then it would become an invalid example

Reply:
okay

Reply:
I think this one still hasnt been fixed